x/concentrated-liquidity/pool_test.go

```diff
@@ -881,8 +876,31 @@ func (s *KeeperTestSuite) TestMigrateAccumulatorToScalingFactor() {
 	// Ensure that position 1 accumulator is not updated (zero)
 	s.validateUptimePositionAccumulator(incentivizedUpdatedAccumulator, positionOneID, cl.EmptyCoins)
 
-	// Ensure that position 1 can claim the same amount as before the migration
-	s.validateClaimableIncentives(positionOneID, claimableIncentivesOneBeforeMigration)
+	// Rerun the same swap to get the same result for the incentive
+	//
+	positionOneCompareID, _ := s.CreateFullRangePosition(concentratedPool, DefaultCoins)
+
+	// Create incentive
+	totalIncentiveAmount = sdk.NewCoin(incentiveDenom, osmomath.NewInt(1000000))
+	s.FundAcc(s.TestAccs[0], sdk.NewCoins(totalIncentiveAmount))
+	_, err = s.App.ConcentratedLiquidityKeeper.CreateIncentive(s.Ctx, poolID, s.TestAccs[0], totalIncentiveAmount, emissionRatePerSecDec, s.Ctx.BlockTime(), types.DefaultAuthorizedUptimes[0])
+	s.Require().NoError(err)
+
+	// Increate block time
+	s.Ctx = s.Ctx.WithBlockTime(s.Ctx.BlockTime().Add(time.Minute))
+
+	// Refetch pool
+	concentratedPool, err = s.App.ConcentratedLiquidityKeeper.GetConcentratedPoolById(s.Ctx, poolID)
+	s.Require().NoError(err)
+
+	// Cross next right tick to update the tick accumulator by swapping
+	amtIn, _, _ = s.computeSwapAmounts(poolID, concentratedPool.GetCurrentSqrtPrice(), currentTick+100, false, false)
+	s.swapOneForZeroRight(poolID, sdk.NewCoin(USDC, amtIn.Ceil().TruncateInt()))
+
+	claimableIncentivesCompareOneAfterMigration, _, err := s.App.ConcentratedLiquidityKeeper.GetClaimableIncentives(s.Ctx, positionOneCompareID)
+
+	// Do the same swap as before the migration to get the same result
+	s.Require().Equal(claimableIncentivesCompareOneAfterMigration.String(), claimableIncentivesOneBeforeMigration.String())
```

Comment:
This is a general repeat of what happens at the start of this test right? Could we use a helper func if so to improve readability?

Reply:
Good idea but this actually took me a long time to refactor... and I couldn't get it to work as expected... :grimacing: